explorer/src/entity/ChainlinkNode.ts

```diff
@@ -175,40 +175,30 @@ export async function uptime(db: Connection, node: ChainlinkNode | number) {
 
 // uptime from completed sessions
 async function historicUptime(db: Connection, id: number): Promise<number> {
-  const { seconds } = await db
+  const { seconds } = (await db
     .createQueryBuilder()
     .select(
-      `FLOOR(SUM(
-        (31536000 * DATE_PART('year', session.finishedAt - session.createdAt)) +
-        (86400 * DATE_PART('day', session.finishedAt - session.createdAt)) +
-        (3600 * DATE_PART('hour', session.finishedAt - session.createdAt)) +
-        (60 * DATE_PART('minute', session.finishedAt - session.createdAt)) +
-        (DATE_PART('second', session.finishedAt - session.createdAt))
-      )) as seconds`,
+      `EXTRACT(EPOCH FROM session."finishedAt" - session."createdAt") as seconds`,
     )
     .from(Session, 'session')
     .where({ chainlinkNodeId: id })
     .andWhere('session.finishedAt is not null')
-    .getRawOne()
-  return parseInt(seconds) || 0
+    // NOTE: If there are no sessions, SELECT EXTRACT... returns null
+    .getRawOne()) || { seconds: 0 }
+  return Math.max(0, seconds)
 }
 
 // uptime from current open session
 async function currentUptime(db: Connection, id: number): Promise<number> {
-  const { seconds } = await db
+  const { seconds } = (await db
```

Comment:
Could we store this query in a temp variable? Wrapping awaits in parentheses for order of execution reduce readability a lot.

```ts
const queriedSeconds = await db.query....
const seconds = queriedSeconds.seconds ?? 0
```